src/actions/app.js

```diff
@@ -75,6 +75,10 @@ export function changeSidebarOpenState(tab: TabSlug, isOpen: boolean): Action {
   return { type: 'CHANGE_SIDEBAR_OPEN_STATE', tab, isOpen };
 }
 
+export function incrementPanelLayoutGeneration(): Action {
```

Comment:
**optional** nit: I'm not especially happy with the name; other similar functions have names that are closer to the intended effect, eg `focusCallTree`.

Maybe `invalidatePanelLayout` (as you called the PR) ? and  `'INVALIDATE_PANEL_LAYOUT'` as action ?

Reply:
Yeah, that's better.